commons/src/main/java/org/infinispan/commons/util/SmallIntSet.java

```diff
@@ -24,33 +24,35 @@
  *
  * @author Dan Berindei
  * @since 9.0
+ * @deprecated since 9.3 This class will no longer be public
  */
-public class SmallIntSet implements IntSet {
+@Deprecated
+public class SmallIntSet implements InternalIntSet {
    private final BitSet bitSet;
 
    public static SmallIntSet of(int i1) {
-      SmallIntSet set = new SmallIntSet();
+      SmallIntSet set = new SmallIntSet(1);
```

Comment:
Maybe `i1 + 1`? Same in the other `of(...)` methods, the constructor parameter is the end of the range.

Reply:
Ah yes, good catch.